include/gz/gui/resources.qrc

```diff
@@ -39,8 +42,11 @@
     <!-- This qmldir file defines a QML module that can be imported -->
     <file alias="qmldir">qml/qmldir</file>
     <!-- Add any QML components referenced in the qmldir file here -->
+    <file alias="GzColor.qml">qml/GzColor.qml</file>
+    <file alias="GzPose.qml">qml/GzPose.qml</file>
     <file alias="GzSnackBar.qml">qml/GzSnackBar.qml</file>
     <file alias="GzSpinBox.qml">qml/GzSpinBox.qml</file>
+    <file alias="GzVector3.qml">qml/GzVector3.qml</file>
     <file alias="IgnSnackBar.qml">qml/GzSnackBar.qml</file>
     <file alias="IgnSpinBox.qml">qml/GzSpinBox.qml</file>
```

Comment:
```suggestion
    <file alias="GzSnackBar.qml">qml/GzSnackBar.qml</file>
    <file alias="GzSpinBox.qml">qml/GzSpinBox.qml</file>
```

Reply:
I believe we can get rid of the duplicate QML files, let's try to get this one first:

* https://github.com/gazebosim/gz-gui/pull/449